testing/cli-e2etest/testscenarios/pollingprofile/list_pollingprofile_test.go

```diff
@@ -95,16 +95,17 @@ func TestListPollingProfile(t *testing.T) {
 		result := tracetestcli.Exec(t, "list pollingprofile --sortBy name --output json", tracetestcli.WithCLIConfig(cliConfig))
 		helpers.RequireExitCodeEqual(t, result, 0)
 
-		pollingProfileYAML := helpers.UnmarshalJSON[[]types.PollingProfileResource](t, result.StdOut)
-
-		require.Len(pollingProfileYAML, 1)
-		require.Equal("PollingProfile", pollingProfileYAML[0].Type)
-		require.Equal("current", pollingProfileYAML[0].Spec.ID)
-		require.Equal("current", pollingProfileYAML[0].Spec.Name)
-		require.True(pollingProfileYAML[0].Spec.Default)
-		require.Equal("periodic", pollingProfileYAML[0].Spec.Strategy)
-		require.Equal("50s", pollingProfileYAML[0].Spec.Periodic.RetryDelay)
-		require.Equal("10m", pollingProfileYAML[0].Spec.Periodic.Timeout)
+		pollingProfilesList := helpers.UnmarshalJSON[types.ResourceList[types.PollingProfileResource]](t, result.StdOut)
```

Comment:
this scenario was incorrect (as well as the old implementation). it was returning an array of resources instead of a `ResourceList`